src/EVM/Fetch.hs

```diff
@@ -235,6 +235,9 @@ oracle solvers info q = do
            Nothing ->
              error ("oracle error: " ++ show q)
 
+    EVM.PleaseDebugMe _vm continue ->
+      pure continue
```

Comment:
This cheat code is useful for programs that use hevm as a library. To consider if it should be enabled for hevm here.